app/lib/dttp/params/placement_assignment.rb

```diff
@@ -55,6 +54,7 @@ def build_params
           "dfe_undergraddegreedateobtained" => Date.parse("01-01-#{trainee.degrees.first.graduation_year}").to_datetime.iso8601,
         }
         .merge(qualifying_degree.uk? ? uk_specific_params : non_uk_specific_params)
```

Comment:
I think we'll also need to move this line into the `requires_degree?` guarded `degree_subject_params`

Reply:
done